### ISSUE TITLE
Please add support for .NET Portable apps

### ISSUE THREAD

**Issue by weitzhandler** [FamilySearch/csharp-lite#4]
Hi,
Please make this project compatible with .NET portable class libraries, so that it can referenced from Xamarin projects or other cross-platform libraries.

**Comment by wooddani** [comment]
I agree the .NET Standard is likely the direction going forward. If you get to it before I do, please submit a pull request.

**Comment by pcolmer** [comment]
In the absence of a version that is compatible with .NET Standard, is there any advice on how to get Gedcomx.Api.Lite(.Core) to work with UWP? Can create a separate issue if required but felt that it was pertinent to this one.

**Comment by wooddani** [comment]
This should be very doable and we are open collaborators. Please submit a
pull request.
